src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/PublicationServiceTests.cs

```diff
@@ -1318,20 +1410,21 @@ await using (var context = InMemoryApplicationDbContext(contextId))
             }
         }
 
-        private static PublicationService BuildPublicationService(ContentDbContext context,
+        private static PublicationService BuildPublicationService(
+            ContentDbContext context,
             IUserService? userService = null,
             IPublicationRepository? publicationRepository = null,
-            IPublishingService? publishingService = null,
-            IMethodologyVersionRepository? methodologyVersionRepository = null)
+            IMethodologyVersionRepository? methodologyVersionRepository = null,
+            IBlobCacheService? publicBlobCacheService = null)
         {
             return new(
                 context,
                 AdminMapper(),
                 new PersistenceHelper<ContentDbContext>(context),
                 userService ?? AlwaysTrueUserService().Object, 
-                publicationRepository ?? new Mock<IPublicationRepository>().Object,
-                publishingService ?? new Mock<IPublishingService>().Object, 
-                methodologyVersionRepository ?? new Mock<IMethodologyVersionRepository>().Object);
+                publicationRepository ?? Mock.Of<IPublicationRepository>(),
+                methodologyVersionRepository ?? Mock.Of<IMethodologyVersionRepository>(),
+                publicBlobCacheService ?? Mock.Of<IBlobCacheService>());
```

Comment:
I think you should use Strict behaviour on all the mocks here like we do elsewhere.